packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -112,6 +118,19 @@ void main() {
         expect(listItems, orderedEquals(<String>['Item 2', 'Item 3', 'Item 4', 'Item 1']));
       });
 
+      testWidgets('properly reorders with a footer', (WidgetTester tester) async {
+        await tester.pumpWidget(build(footer: const Text('Footer Text')));
+        expect(find.text('Footer Text'), findsOneWidget);
```

Comment:
I made a comment about this earlier, but it seems like there is a check here that is missing that would point out a bug in the implementation: 
```
        await longPressDrag(
          tester,
          tester.getCenter(find.text('Item 1')),
          tester.getCenter(find.text('Item 4')) + const Offset(itemHeight * 2, 0.0),
        );
        await tester.pumpAndSettle();
        expect(find.text('Footer'), findsOneWidget);
        expect(listItems, orderedEquals(<String>['Item 2', 'Item 3', 'Item 4', 'Item 1']));
```

In a sample app I created with a ReorderableListView, I cannot manually drag an item from any location to the end of the reorderable list view right before a footer if it exists.

Reply:
I've added the missing part.